### PR TITLE
Car-Rental-Front-End: Car list page

### PR DESCRIPTION
**Pull request by Profsain**
## Summary

- Added Car list page

## Changes log

- [x] Installed React Pagination
- [x] Created Pagination
- [x]  Added car list card

## Acceptance criteria

- [x] The page should have title
- [x] The page should contain car_name, description, car_image
- [x] The page should allow user to move left or right
- [x] When user clicks on specific car, it should redirect to car detail page
- [x] .  Top of the page
carTop
- [x] . Bottom of the page
carBottom

### CODE REVIEW

**Comment by wassimchakib** [comment]
Hi @Profsain,
# STATUS: INVALID REQUEST ❌ ❌ 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that doesn't have a passed linters, please try to fix them 👍 👍 

Cheers and Happy coding!👏👏👏

**Review by KIsmoilov — CHANGES_REQUESTED** [review]
Hi @Profsain    , 

Although you made an effort to implement the requirements, you still need to fix several issues to proceed.

almost4

 ## Highlights 💯 
- No linter errors. :heavy_check_mark:
- Git flow is followed. :heavy_check_mark:
- PR has a good title and summary. ✔️

 ## Required Changes ♻️
- As you can see in the screenshot that the pagination container does not have a nice design. 
- Please fix the width by changing it to 100%
- There should be consistency between the positions of page numbers.
- Please add some margin-top to the title of the page.

Screen Shot 2023-04-11 at 11 23 13 AM (2)

- As you can see in the screenshot when the user scrolls down the pagination container stays fixed in the middle of the screen, please fix it

Screen Shot 2023-04-11 at 11 26 21 AM (2)

- The buttons in the pagination container are having an issue when hovered and clicked.  Please pay attention when the cursor is in the middle of the button it is clickable, but if you put the cursor a little bit closer to the border it is unclickable. So that the border and styling should have been implemented not on the `<li class="page-item>"`, but on the `<a class="page-link"`.

Screen Shot 2023-04-11 at 11 36 05 AM (2)






## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

**Comment by Profsain** [comment]
Hi @KIsmoilov 
I have implemented all required changes. Kindly review my PR

File: src/components/cars-components/cars.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    id: 8,
+    image: 'https://imgd-ct.aeplcdn.com/1056x660/cw/ec/36129/MercedesBenz-G63-AMG-New-Exterior-136618.jpg?wm=0&q=75',
+    name: 'Mercedes-Benz G-Class',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Lorem ipsum dolor sit amet consectetur adipisicing elit. Lorem ipsum dolor sit amet consectetur adipisicing elit. Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+    price: 230,
+    isRented: false,
+  },
+];
+
+export default items;
```
Comment (wassimchakib) [inline src/components/cars-components/cars.js:68]: [Optional], Please try to delete this static JSON file in the future and use redux state

File: src/index.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
+// import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (wassimchakib) [inline src/index.js:4]: [Optional], Kindly delete the comment here

**Review by wassimchakib — APPROVED** [review]
Hi @Profsain , 
I'm Wassim your reviewer for this PR. :wave: :wave: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
:heavy_check_mark: Followed GitFlow.
:heavy_check_mark: No linter errors.
:heavy_check_mark: Good PR summary.

## Optional suggestions
[OPTIONAL] N/A 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**